its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerEngineTest.java

```diff
@@ -147,6 +150,32 @@ void javaExe_withCacheDirectory(boolean useSonarScannerCLI) {
     }
   }
 
+  @ParameterizedTest
+  @MethodSource("parameterizedArgumentsForAbsolutePath")
```

Comment:
Consider using CsvSource instead
https://www.baeldung.com/parameterized-tests-junit-5#4-csv-literals
This is more local than parameterizedArgumentsForAbsolutePath()
```suggestion
  @CsvSource({"sonar.scanner.useSonarScannerCLI,true,SonarScannerCliPath", "sonar.scanner.useSonarScannerCLI,false,EngineJarPath", "sonar.scanner.skipJreProvisioning,false,JavaExePath"})
```